src/main/scala/com/cognite/sdk/scala/v1/fdm/common/properties/PropertyDefinition.scala

```diff
@@ -43,33 +44,64 @@ object PropertyDefinition {
   ) extends ViewPropertyDefinition
       with CorePropertyDefinition
 
-  final case class ConnectionDefinition(
+  sealed trait ConnectionDefinition extends ViewPropertyDefinition
+
+  final case class ReverseDirectRelationConnection(
+      name: Option[String],
+      description: Option[String],
+      connectionType: ReverseDirectRelationConnectionType,
```

Comment:
```suggestion
      connectionType: ReverseDirectRelationConnectionType,
      name: Option[String],
      description: Option[String],
```
just cosmetic option to match docs order and keep type first like in EdgeConnection